StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/AttributedTextView.swift

```diff
@@ -11,7 +11,10 @@ import SafariServices
 @_spi(STP) import StripeUICore
 import UIKit
 
-final class ClickableLabelNew: HitTestView {
```

Comment:
Renamed because `ClickableLabelNew` made me confused: I tried to use it as a `UILabel` replacement, but `UITextView` is multi-line, so its not a direct replacement. `AttributedTextView` also matches with its 'similar' `AttributedLabel`.